### DISCUSSION TITLE
Any good english resources?

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [IrineSistiana/mosdns#332] [General]
I'm sorry to intrude, but I stumbled upon this project, and using google translate - it seems like something I would want to setup.

Is there a written tutorial (one that I can google translate into English) or some other English based reference material?

**Comment by crysislinux** [comment]
Seems there is no English tutorial. The doc is in the wiki: https://github.com/IrineSistiana/mosdns/wiki
the most important entries in the wiki are: https://github.com/IrineSistiana/mosdns/wiki/Configuration and https://github.com/IrineSistiana/mosdns/wiki/Plugin

`mosdns` uses some kind of DSL to customize the DNS resolving.

**Reply by Lockszmith-GH** [reply]
Thanks @crysislinux, I'll try my best with Google Translate - see if I can make sense of it.
